src/components/Output/style.scss

```diff
@@ -15,12 +15,176 @@ Note: These styles are temporary. They will be replaced before going live.
 .output {
   @extend %fill;
 
+  --background-color: rgba(0,0,0,0.05);
+  background-color: #fff;
+  background-repeat: repeat;
+  background-image:
+    linear-gradient(45deg, var(--background-color) 25%, transparent 25%),
+    linear-gradient(-45deg, var(--background-color) 25%, transparent 25%),
+    linear-gradient(45deg, transparent 75%, var(--background-color) 75%),
+    linear-gradient(-45deg, transparent 75%, var(--background-color) 75%);
+  background-size: 20px 20px;
+  background-position: 0 0, 0 10px, 10px -10px, -10px 0px;
+  transition: background-color 300ms ease;
+
+  &.altBackground {
+    background-color: #777;
+  }
+
   > two-up {
     @extend %fill;
 
+    [two-up-handle] {
+      background: #43CEF1;
+      box-shadow: inset 4px 0 0 #34B9EB, 0 1px 4px rgba(0,0,0,0.5);
+      cursor: ew-resize;
+      width: 9px;
+
+      &:after {
+        width: 62px;
+        height: 56px;
+        background: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" width="30" height="20" fill="none"><path fill="#34B9EB" d="M10 0v20L0 10zM20 0v20l10-10z"/></svg>') center no-repeat #fff;
+        border: 1px solid rgba(0,0,0,0.12);
+        border-radius: 5px;
+        box-shadow: 0 1px 4px rgba(0,0,0,0.1);
+        transform-origin: 50% 50%;
+        transition: all 300ms ease;
```

Comment:
I prefer listing out what properties are actually being transitioned. Otherwise you end up doing slow animations without realizing it.

Reply:
Agreed. I've removed the transition anyway.